torch_xla/csrc/tensor.cpp

```diff
@@ -198,7 +198,7 @@ xla::int64 XLATensor::GetUniqueId() const { return data()->unique_id; }
 std::shared_ptr<xla::ComputationClient::Data> XLATensor::GetXlaData() {
   bool up_to_date = true;
   if (data()->view != nullptr) {
-    auto ir_node_updated = GetViewIrNode(data()->view.get());
+    ViewIrNode ir_node_updated = GetViewIrNode(data()->view.get());
```

Comment:
Is the explicit type really needed here? It's pretty obvious what a `GetViewIrNode` is supposed to return.

Reply:
I like to reserve auto for things which are templated.
Having a type explicit avoid looking up the API to figure out.